tests/E2E/scripts/combine-reports.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
```

Comment:
As far as I remember, @eternoendless wasn't okay with this language. This should be discussed to go on this topic.